src/checker.rs

```diff
@@ -301,6 +319,29 @@ mod test {
         assert!(!check(&rules, &String::from("This has a's")));
     }
 
+    #[test]
+    fn test_stem_separator_regex() {
+        let rules : Rules = Rules {
+            stem_separator_regex: "[']".to_string(),
+            disallowed_words: ["Smithsonian", "DC", "Museum"].iter().map(|s| (*s).to_string()).collect(),
+            ..Default::default()
+        };
+
+        assert!(check(&rules, &String::from("The Mall has many museums.")));
+        assert!(!check(&rules, &String::from("Smithsonian's venues are in the Mall.")));
+        assert!(!check(&rules, &String::from("Do you know Smithsonian's African American Museum's location?")));
+        assert!(!check(&rules, &String::from("Washington DC's Mall has many musums.")));
+
+        let rules : Rules = Rules {
+            stem_separator_regex: "[]".to_string(),
```

Comment:
This is an invalid regex, can you replace this test case with a case that tests if it's not provided at all?

Reply:
OK, done.